src/frontend/apps/desk/src/features/auth/useAuthStore.tsx

```diff
@@ -1,40 +1,47 @@
 import { create } from 'zustand';
 
-import { initKeycloak } from './keycloak';
+import { UserData, getMe } from '@/features/auth/api';
 
 interface AuthStore {
   authenticated: boolean;
   initAuth: () => void;
   initialized: boolean;
   logout: () => void;
-  token: string | null;
+  userData?: UserData;
 }
 
 const initialState = {
   authenticated: false,
   initialized: false,
-  token: null,
+  userData: undefined,
 };
 
 export const useAuthStore = create<AuthStore>((set) => ({
   authenticated: initialState.authenticated,
   initialized: initialState.initialized,
-  token: initialState.token,
+  userData: initialState.userData,
 
   initAuth: () =>
     set((state) => {
-      if (process.env.NEXT_PUBLIC_KEYCLOAK_LOGIN && !state.initialized) {
-        initKeycloak((token) => set({ authenticated: true, token }));
-        return { initialized: true };
+      if (state.initialized) {
+        return {};
       }
-
-      /**
-       * TODO: Implement OIDC production authentication
-       */
-
+      getMe()
+        .then((data: UserData) => {
+          set({ initialized: true, authenticated: true, userData: data });
```

Comment:
As it is now, `initialized` and `authenticated` are doing the same job.

Reply:
I reverted it to `set({ authenticated: true, userData: data });` is that okay?